assets/vue/views/accessurl/DeleteAccessUrl.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="space-y-6">
+    <CMAlert type="error" :text="t('Danger zone: deleting an access URL is permanent.')" />
+    <section class="p-0">
+      <h3 class="mb-2 text-sm font-semibold text-rose-900">{{ t("Confirm deletion") }}</h3>
+      <p class="mb-3 text-sm text-rose-800">
+        {{ t("Type the full URL to confirm. The URL and its relations will be permanently removed.") }}
+      </p>
+      <div class="grid grid-cols-1 gap-4 md:grid-cols-2">
+        <div>
+          <label class="mb-1 block text-xs font-medium text-rose-900">{{ t("URL") }}</label>
+          <input
+            v-model="confirmText"
+            class="w-full rounded border border-rose-300 p-2 text-sm"
+            :placeholder="urlText || 'https://example.com'"
+          />
+          <p v-if="confirmText && !canDelete" class="mt-1 text-xs text-rose-700">
+            {{ t("The value must match exactly:") }} <strong>{{ urlText }}</strong>
+          </p>
+        </div>
```

Comment:
Please use the base components for UI
`/assets/vue/components/basecomponents/BaseInputText.vue`